src/customer-service/Actions/CreateIncidentAction.cs

```diff
@@ -0,0 +1,158 @@
+﻿using customer_service.Factories;
+using customer_service.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace customer_service.Actions
+{
+    public class CreateIncidentAction
+    {
+        /**
+         * Class: CreateIncidentAction
+         * Purpose: Prompts the user for information about a new incident and saves that incident to the database
+         * Author: Garrett Vangilder
+         * Methods:
+         *     static void ReadInput() - Prompts for incident info and saves to DB
+         */
+        public CreateIncidentAction()
+        {
+
+        }
+        /**
+        * Purpose: Reads user input for new incident information
+        * Return:
+        *     void
+        */
+        public static void ReadInput()
+        {
+            IncidentFactory incidentFactory = new IncidentFactory();

```

Comment:
It would be more readable to instantiate these factories in the context in which they are used